deps.bzl

```diff
@@ -21,11 +21,12 @@ def gazelle_dependencies():
       commit = "f3dd8fd95a7d078cb10fd7fb475b22c3cdbcb307", # 0.2.0 as of 2017-12-04
   )
 
-  _maybe(go_repository,
-      name = "com_github_pelletier_go_toml",
-      importpath = "github.com/pelletier/go-toml",
-      commit = "16398bac157da96aa88f98a2df640c7f32af1da2", # v1.0.1 as of 2017-12-19
-  )
+  # TODO(jayconrod): restore when gazelle is no longer built for go_repository.
```

Comment:
Seems like the perfect opportunity to try
```
local_repository(
  name = "com_github_pelletier_go_toml",
  path = "endor/github.com/pelletier/go-toml",
)
```

:)

Reply:
Done. I'm pretty surprised this works.